### DISCUSSION TITLE
QSMxT: WARNING: Number of T1w and magnitude runs do not match

### DISCUSSION THREAD

**Discussion by manonlevayer** [QSMxT/QSMxT#34] [Q&A] [answered]
Hi! 

I am currently trying to run the QSMxT application with the demo data supplied on this page https://neurodesk.github.io/tutorials/phase_processing/qsm/ but when I try to use the following command:

python3 /opt/QSMxT/run_3_segment.py 01_bids 03_segmentation

I get the error:

QSMxT: WARNING: Number of T1w and magnitude runs do not match for sub-170705-134431-std-1312211075243167001/ses-1

Just wondering if anyone can help me with that.

Thanks in advance!

**Comment by stebo85** [comment, marked as answer]
Dear @manonlevayer 

The demo data does not come with an anatomical scan, therefore run_3_segment.py will not work.

I now made this clearer in the tutorial. 

All the best
Steffen
